lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -387,8 +406,13 @@ class _RenderIosPagedMenu extends RenderBox
 
   @override
   void performLayout() {
+    // Don't enforce tight constraints on the children.
+    // If we get tight constraints, we still want to let the children
+    // decide its width.
+    final looseConstraints = constraints.loosen();
```

Comment:
Do we want to loosen the height, too? Or just the width?

Reply:
Just the width will work.